p4-16/spec/P4-16-spec.mdk

```diff
@@ -8645,10 +8645,10 @@ are treated as keywords only in specific contexts (e.g., the keyword `actions`).
 | `if`    |   `in`   | `inout` | `int`  |
 | `list`  | `match_kind` | `package` | `parser` |
 | `out`   | `return`  | `select` | `state` |
-| `struct` | `switch` | `table`   | `this` |
-| `transition` | `true` |  `tuple` | `type` |
-| `typedef` | `value_set` | `varbit` | `verify` |
-| `void` | | | |
+| `string` | `struct` | `switch` | `table`   |
+| `this` | `transition` | `true` |  `tuple` |
+| `type` | `typedef` | `value_set` | `varbit` |
+| `verify` | `void` | | |
```

Comment:
For HTML/PDF generation with highlighting, it is useful to ensure that the list of keywords in the file p4.json is consistent with the language, too.

Reply:
the funny thing is that string appears in the json file